package.json

```diff
@@ -43,7 +43,7 @@
     "split": "^1.0.0",
     "srs": "~1.2.0",
     "wmshp": "~0.4.0",
-    "wmtiff": "~0.4.0"
+    "wmtiff": "https://mapbox-npm.s3.amazonaws.com/package/wmtiff-v0.4.0-1-ge6c957b-0813eb808d7b90c233fec3dc5d3e0b87fc15d389.tgz"
```

Comment:
@millzpaugh - please plan to update this before merging by a) tagging a new version of node-wmtiff `v0.4.1`, b)   updating this to `"wmtiff": "~0.4.1"`